octobot_trading/personal_data/orders/order_factory.py

```diff
@@ -79,4 +79,5 @@ def create_order_instance(trader,
                  tag=tag,
                  reduce_only=reduce_only,
                  quantity_currency=quantity_currency)
+    order.ensure_order_id()
```

Comment:
Do we have any idea why a self managed order may not have an id?

Reply:
It's happening all the time on real trading because real trader is not assigning ids to orders, they are fetched from exchange (except those ones)

Reply:
OK I don't understand why we couldn't assign an id at order creation as we already know it's a self managed order (in order factory for example).